src/components/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+import { Link } from "react-router-dom";
+import propTypes from "prop-types";
+
+export const Button = ({
+  to,
+  children,
+  type,
+  styleClass,
+  onClick,
+  disabled,
+}) => {
+  if (to) {
+    return <Link to={to}>{children}</Link>;
+  }
+  return (
+    <button
+      type={type}
+      className={`btn ${styleClass}`}
```

Comment:
I don't see anywhere any class for `Button`. No styles, other than `styleClass`. Besides, I would change `styleClass` to `classes` to be consistent.